src/Language/PureScript/Errors.hs

```diff
@@ -184,6 +185,23 @@ data HintCategory
 
 data ErrorMessage = ErrorMessage [ErrorMessageHint] SimpleErrorMessage deriving (Show)
 
+-- | Get the source span for an error
+errorSpan :: ErrorMessage -> SourceSpan
+errorSpan = fromMaybe (error "errorSpan: no position information") . findHint matchSpan
+  where
+  matchSpan (PositionedError ss) = Just ss
+  matchSpan _ = Nothing
+
+-- | Get the module name for an error
+errorModule :: ErrorMessage -> ModuleName
+errorModule = fromMaybe (error "errorModule: no position information") . findHint matchModule
```

Comment:
Is it safe to do this? I know we _should_ have module info included with every warning and error, but I'm not 100% confident that they all do right now.